src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
`file.h`:
```C++
void f(A a);
```
proper `file.cpp` (does not compile):
```C++
#include "file.h"
#include <a.h>

void f(A a)
{
}
```
Improper `file.cpp` (compiles):
```C++
#include <a.h>
#include "file.h"

void f(A a)
{
}
```
To make the header file complete, one must add `#include <a.h>` to it. This mistake is easily spotted in case of the proper implementation file.